src/test/java/hudson/plugins/cobertura/CoberturaCoverageParserTest.java

```diff
@@ -140,29 +138,8 @@ public void testParseMemoryUsage() throws Exception {
             InputStream in = getClass().getResourceAsStream(fileName);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.

Reply:
This is test code. It's not going to be extended in another package.

Reply:
Thank you for your quick reply. Noticed :)